### PR TITLE
[16.0][MIG] module_auto_update

### PR DESCRIPTION
**Pull request by fkantelberg**
This migrates the module to 16.0. No changes big changes were necessary only a deprecation warning in the flush.

### CODE REVIEW

**Review by hbrunn — APPROVED** [review]

File: module_auto_update/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 LasLabs Inc.
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Module Auto Update",
+    "summary": "Automatically update Odoo modules",
+    "version": "16.0.1.0.1",
```
Comment (yostashiro) [inline module_auto_update/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

**Review by ghost — APPROVED** [review]

**Comment by simahawk** [comment]
Seems we have a flappy test in `base_exception`.

**Comment by hbrunn** [comment]
@fkantelberg could you rebase?

(if you allow edits by maintainers when opening the PR the above command works and things go smoother)

**Comment by fkantelberg** [comment]
@hbrunn rebased and I'll set the flag the next time. Please try again.

**Comment by hbrunn** [comment]
@fkantelberg thanks, and as @yostashiro's comment was fixed too I'll say

/ocabot merge nobump

**Pull request merged**